docker/container/container.go

```diff
@@ -88,3 +93,13 @@ func StartContainer(ctx context.Context, cli *client.Client, name string) entity
 		Type:  entity.FatalType, //todo not sure what to put here yet
 	}
 }
+
+// createCommands is intended to aggregate all necessary commands (including arguments) to run when starting the container
+func createContainerCommands(c entity.Container) []string {
```

Comment:
What does this function do, copy the array?

Reply:
I assumed there may be more commands necessary to add to entrypoint cmd than just `args` so I created a separate function for it so that, if in the future, we decide we need to add more commands to entrypoint, then we can use that function for it.

Reply:
Nope, there arent

Reply:
Please, just use args